assets/js/components/user-input/UserInputKeywords.js

```diff
@@ -43,7 +43,6 @@ const { useSelect, useDispatch } = Data;
 
 export default function UserInputKeywords( { slug, max } ) {
 	const keywordsContainer = useRef();
-	const [ canDeleteKeyword, setCanDeleteKeyword ] = useState( false );
```

Comment:
@danielgent why did you delete this?

Reply:
@eugene-manuilov This file really confused me. I'm happy to revert it all and just disable eslint for the file :smile: 

This state didn't appear to do anything. `canDeleteKeyword` was only used for unnecessary dependencies.

the react hooks eslint plugin forced me to remove it as a dependency, and then linting forced me to delete the variable due to it being unused!